contrib/cf-remote/cf_remote/commands.py

```diff
@@ -64,6 +64,41 @@ def scp(hosts, files, users=None):
             errors += transfer_file(host, file, users)
     return errors
 
+def _download_urls(packages):
```

Comment:
I would maybe rename argument to `urls` here (but don't change how this function is called) - it would probably make the function more readable.

Reply:
https://github.com/cfengine/core/pull/4343